plugins/ssl/letsencrypt_weekly

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# monitor the number of CSRs by week for /etc/letsencrypt/csr/
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Parameters:
+#
+#  config
+#  autoconfig
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+if [ "$1" == "autoconf" ] ; then
+    find /etc/letsencrypt/csr/ > /dev/null 2>&1 && echo "yes" || echo "no"
```

Comment:
Please add a reason to `no` (e.g. `no (no files below /etc/letsncrypt/csr)`).

Additionally you should not let the error message of `find /etc/letsencrypt/csr/` (in case of non-existing directory) slip through to the user. Instead you could simply check, whether the directory exists.

Reply:
Improved the `no`

The additional error message is suppressed with `> /dev/null 2>&1` already (Did test that no error message is given to the user when the directory does not exist, only the `no ...` message is printed) .